src/util/apiHelpers.ts

```diff
@@ -130,3 +138,37 @@ export const createApolloClient = (language = 'nb') => {
 
   return client;
 };
+
+type HttpHeaders = {
+  headers?: {
+    'Content-Type': string;
+  };
+};
+
+export const fetchAuthorized = (url: string, config?: HttpHeaders) =>
+  fetchWithAuthorization(url, false, config);
+
+export const fetchWithAuthorization = async (
+  url: string,
+  forceAuth: boolean,
+  config?: HttpHeaders,
+) => {
+  if (forceAuth || !isAccessTokenValid()) {
+    await renewAuth();
+  }
+
+  const contentType = config?.headers
+    ? config?.headers['Content-Type']
+    : 'text/plain';
+  const extraHeaders = contentType ? { 'Content-Type': contentType } : {};
+  const cacheControl = { 'Cache-Control': 'no-cache' };
+
+  return fetch(url, {
+    ...config,
+    headers: {
+      ...extraHeaders,
+      ...cacheControl,
+      Authorization: `Bearer ${getAccessToken()}`,
```

Comment:
Denne må også legges inn på linje 104, for det er der alle kall på graphql-api trigges fra. I tillegg må du bruke navnet FeideAuthorization.
```suggestion
      FeideAuthorization: `Bearer ${getAccessToken()}`,
```